src/js/components/award/shared/additionalInfo/AdditionalInfo.jsx

```diff
@@ -141,6 +153,18 @@ export default class AdditionalInfo extends React.Component {
     faColumns() {
         const { overview } = this.props;
         const data = this.data();
+
+        const cdPOP = data.placeOfPerformance.Address.data.pop().trim();
+        const splitCDPOP = cdPOP.split(": ");
+        if (splitCDPOP.length === 2) {
+            data.placeOfPerformance["Congressional District"] = splitCDPOP[1];
+        }
+
+        const cdRD = data.recipientDetails["Congressional District"].data.pop().trim();
```

Comment:
When this block is left in, it looks like the CD listed here is actually the Place of Performance CD; if it's commented out, the CD listed here changes to match the Recipient Address, but it's formatting is different than the CD in the Place of Performance section; I was using 
CONT_AWD_DEAC0494AL85000_8900_-NONE-_-NONE-
to test.